src/styles/blocks/header.scss

```diff
@@ -1,3 +1,5 @@
+@import "utils/variables.scss";
+
```

Comment:
Move `box-sizing: border-box;` to * selector and remove it from all other selectors.